tools/review_pull_request.py

```diff
@@ -0,0 +1,221 @@
+            full_pull_request.mergeable
+        ):
+            if dry_run:
+                print('Pull request "{}" ({}) ready to merge'.format(
+                    pull_request.title,
+                    pull_request.html_url
+                ))
+            else:
+                wrap_github_call(
+                    pull_request.issue().add_labels,
+                    ['merge ready']
+                )
+                wrap_github_call(
+                    pull_request.create_comment,
+                    ["Ready to merge"]
+                )
+
+
+def tag_all_ready_pull_requests(dry_run=False):
+    for repository_name in OCA_REPOSITORY_NAMES:
+        tag_ready_pull_requests(repository_name, dry_run=dry_run)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--tag-new-pull-requests",
+        help="Tag open pull requests made by new contributors in the given "
+             "repositories.\n"
+             "Contributors are considered new if they have fewer than 3 "
+             "commits in the master branches of all OCA repositories.",
+        nargs="+",
+        default=(),
+        metavar="repository-name"
+    )
+    parser.add_argument(
+        "--tag-all-new-pull-requests",
+        help="Tag open pull requests made by new contributors in all "
+             "repositories.",
+        action="store_true"
+    )
+    parser.add_argument(
+        "--close-abandoned-pull-requests",
+        help="Close abandoned pull requests in the given repositories.\n"
+             "Pull requests are considered abandoned if they have no "
+             "comments, and are over 6 months old",
+        nargs="+",
+        default=(),
+        metavar="repository-name"
+    )
+    parser.add_argument(
+        "--close-all-abandoned-pull-requests",
+        help="Close abandoned pull requests in all repositories.",
+        action="store_true"
+    )
+    parser.add_argument(
+        "--tag-ready-pull-requests",
+        help="Tag open pull requests that are ready to merge in the given "
+             "repositories.\n"
+             "Pull requests are considered ready to merge with two approvals, "
+             "green CI, and have been open at least 5 days",
+        nargs="+",
+        default=(),
+        metavar="repository-name"
+    )
+    parser.add_argument(
+        "--tag-all-ready-pull-requests",
+        help="Tag open pull requests that are ready to merge in all "
+             "repositories.",
+        action="store_true"
+    )
+    parser.add_argument(
+        "--dry-run",
+        help="Print the actions that this command would run, "
+             "instead of applying them.",
+        action="store_true"
+    )
+    args = parser.parse_args()
+
+    for repository_name in args.tag_new_pull_requests:
+        tag_new_pull_requests(repository_name, dry_run=args.dry_run)
+
+    if args.tag_all_new_pull_requests:
+        tag_all_new_pull_requests(dry_run=args.dry_run)
```

Comment:
if you pass `args` to the functions, it's much simpler to implement my proposals above